src/config/setshortcutwidget.cpp

```diff
@@ -32,13 +32,16 @@ SetShortcutDialog::SetShortcutDialog(QDialog* parent)
 
     m_layout->addWidget(infoIcon);
 
+    QString msg = "";
 #if defined(Q_OS_MAC)
-    QLabel* infoBottom = new QLabel(tr(
-      "Press Esc to cancel or ⌘+Backspace to disable the keyboard shortcut."));
+    msg = tr("Press Esc to cancel or ⌘+Backspace to disable the keyboard shortcut.");
 #else
-    QLabel* infoBottom = new QLabel(
-      tr("Press Esc to cancel or Backspace to disable the keyboard shortcut."));
+    msg = tr("Press Esc to cancel or Backspace to disable the keyboard shortcut.");
 #endif
+    if (shortcutName == "TAKE_SCREENSHOT" || shortcutName == "SCREENSHOT_HISTORY"){
+        msg += "\n" + tr("Require restart flameshot"); 
```

Comment:
IMO the message should be changed to "Flameshot must be restarted for changes to take effect.".